deps/check_latest_v8_version.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python3
+
+# Script to compare current v8 version installed and the latest stable v8 version
+# Returns 1 if the versions are the same
+# Returns 0 if the latest stable version is higher than the current installed one
+
+import requests
+import sys
+
+# api-endpoint
+URL = "https://omahaproxy.appspot.com/all.json?os=linux&channel=stable"
+
+def compare_v8_version(current_version, new_version):
+  if current_version == new_version:
+    return 1
+  else:
+    return 0
+
+# Current version
+v8_version_file = open("v8_version", "r")
+current_v8_version_installed = v8_version_file.read().strip()
+
+# Get latest version
+r = requests.get(url = URL)
+
+data = r.json()
+
+latest_stable_v8_version = data[0]["versions"][0]["v8_version"].strip()
+
+result = compare_v8_version(current_v8_version_installed, latest_stable_v8_version)
+sys.exit(result)
```

Comment:
Is this going to make it seem like the github action failed just because there is nothing to do?  It seems like that would make it harder to tell when the github action is actually broken

Reply:
Instead, I think this could be merged update_include_folder.py the update_v8_src.py script and could take on the work from several github action steps, making it easy to skip unnecessary following work and just not change any files.  The [peter-evans/create-pull-request](https://github.com/peter-evans/create-pull-request) github action documents that

> If there are no changes (i.e. no diff exists with the checked-out base branch), no pull request will be created and the action exits silently.

so we wouldn't even have to worry about it creating an empty PR.